src/examples/fake_device_demo.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
+
+require "storage/fake_probing.rb"
+require "storage/fake_device_factory.rb"
+
+FILENAME = "fake-devicegraphs"
+
+fake_probing = Yast::Storage::FakeProbing.new
+devicegraph = fake_probing.devicegraph
+factory = Yast::Storage::FakeDeviceFactory.new(devicegraph)
+factory.load_yaml_file("#{FILENAME}.yml")
+
+fake_probing.to_probed
+puts("Disks:")
+probed = Yast::Storage::StorageManager.instance.probed
+fake_probing.dump_disks(probed)
```

Comment:
Anyway, let's merge this and move forward for the time being. The code works and we need to have all the code merged if we plan to move it to a different repo.

After the sprint is finished, we will decide if this kind of discussions about API belong to code review or not. https://trello.com/c/lTZScC5Z/145-how-to-solve-code-review-disagreements